modules/netstat/netstat.go

```diff
@@ -62,6 +63,7 @@ type element struct {
 	LocalPort     float64 `json:"localport,omitempty"`
 	RemoteAddr    string  `json:"remoteaddr,omitempty"`
 	RemotePort    float64 `json:"remoteport,omitempty"`
+	Namespace     string  `json:"namespace,omitempty"`
```

Comment:
It seems to be a uint value. any reason to store it as a string?
`NamespaceId` would be more explicit.